samples/monster_generated.h

```diff
@@ -510,6 +511,14 @@ inline bool VerifyEquipment(flatbuffers::Verifier &verifier, const void *obj, Eq
   }
 }
 
+inline bool VerifyEquipmentVector(flatbuffers::Verifier &verifier, const flatbuffers::Vector<flatbuffers::Offset<void>> *values, const flatbuffers::Vector<uint8_t> *types) {
+  assert(values->size() == types->size());
```

Comment:
The task of the verifier is to find out if a buffer is safe or corrupt, and do so quickly, and by, default, without asserts. If a buffer is corrupt, that could have all sorts of reasons, including bad transmissions etc, so errors are not necessarily that useful.

But yes, rather than just return, you should wrap the condition in `Verifier::Check`, which allows anyone to debug a verifier issue.